plugin/utility/src/main/java/org/opencds/cqf/ruler/plugin/utility/OperatorUtilities.java

```diff
@@ -0,0 +1,139 @@
+package org.opencds.cqf.ruler.plugin.utility;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This interface provides utilities for implementing FHIR operators.
+ */
+public interface OperatorUtilities extends ResolutionUtilities {
+
+    public static final Logger ourLog = LoggerFactory.getLogger(OperatorUtilities.class);
+
+    /**
+     * This function converts a string representation of a FHIR period date to a
+     * java.util.Date.
+     * 
+     * @param date  the date to convert
+     * @param start whether the date is the start of a period
+     * @return the FHIR period date as a java.util.Date type
+     */
+    public default Date resolveRequestDate(String date, boolean start) {
```

Comment:
This code already exists in the cql-evaluator

Reply:
I looked in the cql-evaluator and couldn't find it.